lighthouse-core/lib/stack-collector.js

```diff
@@ -39,7 +39,7 @@ const libDetectorSource = fs.readFileSync(
 /**
  * Obtains a list of detected JS libraries and their versions.
  */
-/* istanbul ignore next */
+/* c8 ignore start */
```

Comment:
Does this one need a corresponding "c8 ignore stop"?

Reply:
done, good catch